pwiz_tools/Skyline/Skyline.resx

```diff
@@ -3846,4 +3846,11 @@
   <data name="&gt;&gt;ionTypesContextMenuItem.Type" xml:space="preserve">
     <value>System.Windows.Forms.ToolStripMenuItem, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </data>
+  <data name="ColonSpace" xml:space="preserve">

```

Comment:
Seems like these should go with Colon in Resources.resx rather than in Skyline.resx